src/rest-server/src/models/v2/group.js

```diff
@@ -100,26 +100,25 @@ const getGroupsVCs = async (grouplist) => {
   return getVCsWithGroupInfo(groupItems);
 };
 
-const getStorageConfigsWithGroupInfo = async (groupItems) => {
+const getStorageConfigsWithGroupInfo = async (groupItems, filterDefault=false) => {
   let storageConfigs = new Set();
   for (const groupItem of groupItems) {
     if (groupItem.extension && groupItem.extension.acls) {
       if (groupItem.extension.acls.storageConfigs) {
-        storageConfigs = new Set([...storageConfigs, ...groupItem.extension.acls.storageConfigs]);
+        if (filterDefault) {
+          storageConfigs.add(groupItem.extension.acls.storageConfigs[0]);
+        } else {
+          storageConfigs = new Set([...storageConfigs, ...groupItem.extension.acls.storageConfigs]);
```

Comment:
Use 
```js
groupItem.extension.acls.storageConfigs.forEach(item=>storageConfigs.add(item))
```
Instead of create a set each loop?

Reply:
It exists in previous code, will update

Reply:
updated